### DISCUSSION TITLE
Inventories of other worlds

### DISCUSSION THREAD

**Discussion by masterhacka** [Zrips/CMI#3389] [Enhancement]
**Question: **
Hi there,

is there a way to open inventories of other worlds (when using MultiInv or Multiverse-Inventories)?
Otherwise it would be cool if it is possible to implement such a feature.

**Comment by CloudeLecaw** [comment]
No? there is not

For a implemt of a feature like this CMI would need an optional hook for plugins like Multiinv or PerWorldInventory, Multiverse-Inventories an may others..

at the moment it looks for the inventory save in the normal world folder i think uso u just see the inventory that is "currently" activ

**Comment by masterhacka** [comment]
Then it would be nice to implement this feature in the future. A GUI is way better than open up json files for inventories. CMI has a few hooks and to add the more recent ones from the three mentioned  inventory plugins would be sufficient.

Otherwise there is no other way to search for a specific item in different world inventories.

**Comment by CloudeLecaw** [comment]
I agree with you, a nice idea :)
